src/os/track.js

```diff
@@ -1158,6 +1158,41 @@ os.track.promptForTitle = function(opt_default) {
 };
 
 
+/**
+ * Prompt the user to choose a track title, and if original metadata should be included.
+ *
+ * @param {string=} opt_default The default title. Defaults to an empty string.
+ * @param {boolean=} opt_includeMetadata If metadata should be included. Defaults to false.
+ * @return {!goog.Promise}
+ */
+os.track.promptForTitleAndMetadata = function(opt_default = '', opt_includeMetadata = false) {
```

Comment:
Could this get merged into the existing function? I don't think anything else is calling the old one, and that function is already being used for the deprecated `plugin.track.promptForTitle` from when tracks were a plugin, could be confusing to have all these functions doing very close to the same thing.

Reply:
I tried doing that, but that would require passing in the checkbox callback in addition to the default value in order to avoid the promise resolution API change. I didn't really like that solution, but I can go back to that if we want. Or I can deprecate the original function in favor of this one. I was trying to avoid passing around callbacks because the caller only cares about the end result.